src/main.rs

```diff
@@ -351,13 +357,6 @@ impl Numpad {
                     Some(key) => {
                         self.grab();
                         self.state.finger_state = FingerState::Touching;
```

Comment:
Removing this code seems wrong. Now we can't hold a key to repeat it. 

> By default, when you hold down a key on your keyboard, the letter or symbol will be repeated until you release the key

(Gnome docs)